### PR TITLE
Fixes pai don't ask me again (I think)

### LINKED ISSUE #15301: The "never for this round" pAI request is broken
You hit it but it still gives you prompts

### PR DESCRIPTION
**Pull request by MrsTonedOne**
fixes #15301 (I think)

The way it did it here was weird and i think it caused the bug because it would check the ask time + 600 and that would wrap around.

I don't know for sure if that's the cause of the bug, but this is an improvement none the less.

### CODE REVIEW

**Comment by neersighted** [comment]
```
code/controllers/subsystem/pai.dm:211:error: prefs.be_special: undefined var
code/controllers/subsystem/garbage.dm:101:warning: if: if statement has no effect
```

**Comment by GunHog** [comment]
You can use the guest login trick to have two clients open on your private server - That is very useful when you need to test things like this.

File: code/controllers/subsystem/pai.dm [hunk 1]
```diff
@@ -197,18 +197,18 @@ var/datum/subsystem/pai/SSpai
...
+		return
+	asked.Add(C.key)
+	asked[C.key] = world.time
+	var/response = tgalert(C, "Someone is requesting a pAI personality. Would you like to play as a personal AI?", "pAI Request", "Yes", "No", "Never for this round", StealFocus=0, Timeout=askDelay)
+	if(!C)
+		return		//handle logouts that happen whilst the alert is waiting for a response.
+	if(response == "Yes")
+		recruitWindow(C.mob)
+	else if (response == "Never for this round")
+		C.prefs.be_special -= ROLE_PAI
```
Comment (tkdrg) [inline code/controllers/subsystem/pai.dm]: Won't this be persistent, though? Since it would be going into savefiles etc.

Comment (MrStonedOne) [inline code/controllers/subsystem/pai.dm]: point, a lot of things change the save file enough to trigger a save.

Comment (RemieRichards) [inline code/controllers/subsystem/pai.dm]: Well this is only "Never for this round", not "Never, forever, leave me alone pAIs"

Comment (MrStonedOne) [inline code/controllers/subsystem/pai.dm]: I was thinking of character saves when i made this, which are only saved if you hit the save button in setup character.

**Pull request merged**